### DISCUSSION TITLE
cant get chia to work since upgrade

### DISCUSSION THREAD

**Discussion by geordiepaul101** [Chia-Network/chia-blockchain#11776] [Support]
Hi, I was using chia 2.something and it worked fine, just upgraded to 1.3.5 and now at the top right it says sync'd and connected (5) but under "full node" its saying "syncing 0/2068575" in orange, but it doesnt do anything. The sync just stays at zero.

The port is open (I think)... it worked before, any ideas? Thanks

**Comment by loppefaaret** [comment]
we might need to ask you to add your `debug.log` to this discussion, which can be found under your users home folder, in `~\.chia\mainnet\log\`
just drag the file into the message box here, and attach it
